src/objects/index.ts

```diff
@@ -1,25 +1,27 @@
-import { createApplication, application } from '@/objects/application'
-import { createBox } from '@/objects/box'
-import { createScales } from '@/objects/scales'
-import { setStage } from '@/objects/stage'
-import { createViewport } from '@/objects/viewport'
+import { startApplication, stopApplication } from '@/objects/application'
+import { startBox } from '@/objects/box'
+import { emitter } from '@/objects/events'
+import { startScales } from '@/objects/scales'
+import { startStage, stopStage } from '@/objects/stage'
+import { startViewport } from '@/objects/viewport'
 
 export * from './application'
 export * from './stage'
 export * from './viewport'
 export * from './scales'
 
 export function start(stage: HTMLDivElement): void {
-  setStage(stage)
+  startApplication()
+  startViewport()
+  startScales()
+  startBox()
 
-  createApplication()
-  createViewport()
-  createScales()
-  createBox()
+  startStage(stage)
```

Comment:
This PR fixed some order dependent code. Now this `startStage` function just must run last. The others can be run in any order. I'd like this to not matter either but for now this is simplest solution. Adding the stage kicks off the rest of the logic.